data/recommendation/style.scss

```diff
@@ -34,18 +34,16 @@ body {
     width: 100%;
   }
   .image-box {
-    width: 10%;
+    align-items: center;
+    display: flex;
```

Comment:
`margin: auto` fixed horizontal alignment, but what about vertical alignment? Here's how it currently looks: 

<img width="1192" alt="screen shot 2016-08-02 at 11 03 31 am" src="https://cloud.githubusercontent.com/assets/12304716/17341664/82d082ee-58aa-11e6-9f55-47b6e4ff4090.png">
I'd like to have it centered vertically